pkg/operator/operator.go

```diff
@@ -21,28 +23,105 @@ const instanceSelector = "app.kubernetes.io/managed-by=observability-operator"
 
 const ObservabilityOperatorName = "observability-operator"
 
+// The default values we use. Prometheus and Alertmanager are handled by
+// prometheus-operator. For thanos we use the default version from
+// prometheus-operator.
+var DefaultImages = map[string]string{
+	"prometheus":   "",
+	"alertmanager": "",
+	"thanos":       "quay.io/thanos/thanos:" + obopo.DefaultThanosVersion,
```

Comment:
nit: Could we make `quay.io/thanos/thanos`a var? I know we're not using it anymore around but still.

Reply:
How about we use `obopo.DefaultThanosImage` :D